rest/client.go

```diff
@@ -49,3 +54,63 @@ func (c *client) GetComponentsByAsset(ctx context.Context, id uuid.UUID, filter
 
 	return response.Components, nil
 }
+
+func (c *client) GetComponentRelations(ctx context.Context, id uuid.UUID, limit int, continuationToken string) (models.GetComponentRelationsResponse, error) {
```

Comment:
We usually do not expose the paging operation in the clients.

So GetComponentRelations/GetRelatiedComponents should probably just return all and use the pagination if needed. This avoids everyone needing to write the pagination logic on their end.

Reply:
Makes sense - updated.